gm4_audere_shamir/data/gm4_audere_shamir/functions/active_tool.mcfunction

```diff
@@ -0,0 +1,17 @@
+#run from main
+#@s = players holding an Audere tool
+
+scoreboard players set tool_current_damage gm4_ml_data 0
+scoreboard players set tool_max_damage gm4_ml_data 0
```

Comment:
Leaving these two resets in may still be a good idea in case Moneo also relies on these values; referring to:

> In Gamemode 4 we usually reset scores at the end of a clock cycle, this makes these values invisible to players when the scoreboard is shown.
Prefixing fake players with `$` is also recommended.